docs/matlab_differences.rst

```diff
@@ -0,0 +1,103 @@
+.. _matlab-diffs:
```

Comment:
You have this warning in the CI build of the docs:

> /home/runner/work/pyprep/pyprep/docs/matlab_differences.rst: WARNING: document isn't included in any toctree

That can be quenched by:

```suggestion
:orphan:

.. _matlab-diffs:
```